source/beamline/AMPVControl.cpp

```diff
@@ -249,6 +250,13 @@ AMControl::FailureExplanation AMPVControl::move(double setpoint) {
 			return NotConnectedFailure;
 		}
 		setpoint_ = setpoint;
+
+		// Update the move progress values.
+
+		updateMoveStart();
```

Comment:
Well, it might be that AMControl only has a percentage and everyone else has the responsibility of turning start and end points to percentages.